daemon/k8s_watcher.go

```diff
@@ -949,113 +947,6 @@ func (d *Daemon) EnableK8sWatcher(queueSize uint) error {
 		}
 	}()
 
-	asyncControllers.Add(1)
```

Comment:
The cost is very high. What use case would justify using regular nodes instead of CiliumNodes?

Reply:
My thought is that we are moving to a mode which is only truly able to be tested via deployment in a cluster / integration testing, so having an easy way to fall back on existing behavior which we have tested before in case there is a bug by using `CiliumNode` is preferable than just deleting the old code outright. Once we have confidence in the new `CiliumNodes` mode we could delete the old K8s `Node` resource handling.

Reply:
I don't think we have a safe fall back other than kvstore and that still exists. kvstore will still take precedence as it did before. Node discovery on standard k8s resources is not really an alternative as it does not provide encryption information.

Reply:
OK, makes sense - it's an all-or-nothing thing since encryption is involved.

Reply:
Given the struggle we had with etcd-operator. My proposal is to focus all testing this week on gaining as much confidence as possible with this as we will rely on it replacing etcd-operator for simplicity while we refer to external etcd as the go-to scale option as we did before.